File: ExemploApiLocationV/app/src/main/java/nuvemapp/com/br/exemploapilocationv/service/Place.java
package nuvemapp.com.br.exemploapilocationv.service;


import nuvemapp.com.br.exemploapilocationv.R;

/**
 * Created by viniciusthiengo on 3/22/15.
 */
public class Place {
    private String name;
    private int category;
    private String description;
    private double latitude;
    private double longitude;


    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getCategory() {
        return category;
    }

    public void setCategory(int category) {
        this.category = category;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }



    public static int getCategorySource(int c) {
        switch ( c ){
            case 1:
                return(R.drawable.pub);
            case 2:
                return(R.drawable.university);
            case 3:
                return(R.drawable.gas_station);
            case 4:
                return(R.drawable.parking);
            case 5:
                return(R.drawable.hospital);
            case 6:
                return(R.drawable.bakery);
            case 7:
                return(R.drawable.restaurant);
            case 8:
                return(R.drawable.mall);
            default:
                return(R.drawable.grocery_store);
        }
    }
}

File: ExemploApiLocationV/PHP/location-api/package/cdp/Place.php
<?php
	@include_once('config/config.php');
	@include_once('../../config/config.php');
	require_once(__PATH__.'/package/extras/class/Obj.php');
	require_once(__PATH__.'/package/cdp/Category.php');

	
	class Place extends Obj {
		private $name;
		private $category;
		private $description;
		private $latitude;
		private $longitude;
		private $distance;

		
		public function __construct($id=0, $name='', $category=NULL, $description='', $latitude=0, $longitude=0, $distance=0, $status=0, $time=0){
			$this->id = $id;
			$this->name = $name;
			$this->category = $category;
			$this->description = $description;
			$this->latitude = $latitude;
			$this->longitude = $longitude;
			$this->distance = $distance;
			$this->status = $status;
			$this->time = $time;
		}
		public function __destruct(){}
		
		
		public function post($post){
			$this->setName( $post['name'] );
			$this->setCategory( new Category($post['category']) );
			$this->setDescription( $post['description'] );
			$this->setLatitude( $post['latitude'] );
			$this->setLongitude( $post['longitude'] );
			$this->setStatus( $post['status'] );
			$this->setTime( time() );
		}
		
		
		public function getName(){
			return($this->name);
		}
		public function setName($n){
			$this->name = $n;
		}


		public function getCategory(){
			return($this->category);
		}
		public function setCategory($c){
			$this->category = $c;
		}


		public function getDescription(){
			return($this->description);
		}
		public function setDescription($description){
			$this->description = $description;
		}
		
		
		public function getLatitude(){
			return($this->latitude);
		}
		public function setLatitude($latitude){
			$this->latitude = $latitude;
		}
		
		
		public function getLongitude(){
			return($this->longitude);
		}
		public function setLongitude($longitude){
			$this->longitude = $longitude;
		}


		public function getDistance(){
			return($this->distance);
		}
		public function setDistance($distance){
			$this->distance = $distance;
		}

		
		// JSON
			public function getDataJSON(){
				$aux = array(
					'id'=>					$this->getId(),
					'name'=>				$this->getName(),
					'category'=>			is_object($this->getCategory()) ? $this->getCategory()->getDataJSON() : NULL,
					'description'=>			$this->getDescription(),
					'latitude'=>			$this->getLatitude(),
					'longitude'=>			$this->getLongitude(),
					'distance'=>			$this->getDistance(),
					'status'=>				$this->getStatus(),
					'timeMilliseconds'=>	$this->getTimeMilliseconds());
				$aux = $this->setArrayToUtf8($aux);
				return($aux);
			}
	}
?>